gpflow/training/tensorflow_optimizer.py

```diff
@@ -95,6 +95,8 @@ def minimize(self, model, session=None, var_list=None, feed_dict=None,
             initialized before for gotten session.
         :param anchor: If `True` trained variable values computed during optimization at
             particular session will be synchronized with internal parameter values.
+        :param step_callback: A callback function to execute at each optimization step,
+        supplied with the current value of the packed variable vector.
```

Comment:
Minor: you forgot indentation after newline.

Reply:
Fixed